lib/etl/extract/process_files_in_s3_bucket_folders.rb

```diff
@@ -0,0 +1,52 @@
+require 'aws-sdk'
+require 'tempfile'
+
+class ProcessFilesInS3BucketFolders
+  def initialize(bucket_name, prefix_folders, file_ext)
+    @bucket_name = bucket_name
+    @prefix_folders = prefix_folders
```

Comment:
It's an array for convenience, to make the .etl files powerful and tidy.

Right now in lib/etl/process_s3_json_resources.etl  and lib/etl/process_s3_json_groups.etl respectively I'm just pulling all resources and groups.  I.e., paths 'valid_data/resources/' and 'valid_data/groups/' can slurp up all the resources all the groups, because of how data is structured in S3.  However, we might want to use just certain folders, and I wanted that to be easy and terse to do.